### PR TITLE
Added data stores and DM examples

### PR DESCRIPTION
**Pull request by PaulAsjes**
I wrote the new examples in ES6, but happy to either rewrite in ES5 or port the other examples to ES6.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +103,59 @@ rtm.on(RTM_CLIENT_EVENTS.RTM_CONNECTION_OPENED, function () {
...
+  // Boolean indicating whether each message should be marked as read or not after it is processed
+  autoMark: true
+});
+
+rtm.start();
+
+// Wait for the client to connect
+rtm.on(CLIENT_EVENTS.RTM.RTM_CONNECTION_OPENED, () => {
+  // Get the user's name
+  let user = slack.dataStore.getUserById(rtm.activeUserId);
```
Comment (ekmartin) [inline README.md]: Shouldn't `slack` be `rtm` here?

Comment (PaulAsjes) [inline README.md]: Whoops, thanks for catching!

**Comment by l12s** [comment]
Hey, sorry for the slow reply. I'm going to ask @DEGoodmanWilson to take a look at this

**Comment by l12s** [comment]
Ok, I think Don's busy, so I'll take a look. Apologies for the delay.

If you don't mind, I'd prefer to keep everything ES5 until we make the cutover in the code itself. Otherwise, looks great.

**Comment by PaulAsjes** [comment]
@l12s No worries, I switched everything to ES5.

**Comment by l12s** [comment]
Thanks!

**Pull request merged**